app/controllers/application_controller.rb

```diff
@@ -54,6 +66,10 @@ def load_challenge
     @challenge = Challenge.find(params[:challenge_id])
   end
 
+  def default_url_options(options = {})
```

Comment:
The only place that this function is used is with the mailers.  Are the emails also meant to support I18N?